app/src/main/java/org/wikipedia/readinglist/ReadingListBookmarkMenu.java

```diff
@@ -82,16 +86,16 @@ private void showMenu() {
             menu.setGravity(Gravity.END);
 
             MenuItem addToOtherItem = menu.getMenu().findItem(R.id.menu_add_to_other_list);
-            addToOtherItem.setVisible(listsContainingPage.size() > 0);
-            addToOtherItem.setEnabled(listsContainingPage.size() > 0);
+            addToOtherItem.setVisible(isNotEmpty(listsContainingPage));
```

Comment:
I could see perhaps changing this to `!list.isEmpty()`, but what is the benefit of using a new third-party dependency for this?

Reply:
Same as earlier: that list is nullable, so calling its `size()` method might throw a `NullPointerException`.